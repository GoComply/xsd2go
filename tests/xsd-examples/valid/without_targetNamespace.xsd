<?xml version="1.0" encoding="utf-8"?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
    <!-- definition of simple elements -->
    <xs:element name="name" type="xs:string"/>
    <!-- definition of complex elements -->
    <xs:element name="referencingthename">
        <xs:complexType>
            <xs:sequence>
                <xs:element ref="name"/>
            </xs:sequence>
        </xs:complexType>
    </xs:element>
</xs:schema>
