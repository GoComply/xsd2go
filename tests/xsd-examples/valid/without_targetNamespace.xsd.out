// Code generated by https://github.com/gocomply/xsd2go; DO NOT EDIT.
// Models for without_targetNamespace
package without_targetNamespace

import (
	"encoding/xml"
)

// Element
type Name struct {
	XMLName xml.Name `xml:"name"`

	Text string `xml:",chardata"`
}

// Element
type Referencingthename struct {
	XMLName xml.Name `xml:"referencingthename"`

	Name string `xml:",any"`
}

// XSD ComplexType declarations

// XSD SimpleType declarations
